Sources/ProjectSpec/Target.swift

```diff
@@ -2,6 +2,22 @@ import Foundation
 import xcproj
 import JSONUtilities
 
+public struct LegacyTarget {
+    public var buildToolPath: String
+    public var buildArgumentsString: String?
```

Comment:
Does this need a `String` appendix?

Reply:
Another way of naming this could be `LegacyBuild` with the following properties:

- toolPath
- arguments
- passSettings
- workingDirectory

What do you think?